app/static/js/ferm_graph_socketio.js

```diff
@@ -9,7 +9,7 @@ chart: {
   events: {
     load: function () {
       var self = this
-      var socket = io.connect('http://' + document.domain + ':' + location.port)
+      var socket = io.connect('//' + document.domain + ':' + location.port)

```

Comment:
this will allow the browser/client to determine the scheme to use when connecting to `document.domain` keeping https -> https and http -> http instead of downgrading (aka `Mixed Content`) from https -> http which would be blocked client side.

![image](https://user-images.githubusercontent.com/2158627/85853970-7af8a080-b781-11ea-946a-32ad1df19956.png)